Backend/dev/migrations/rider-app/1110-add-column-merchant-service-usage-config.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE atlas_app.merchant_service_usage_config ADD COLUMN get_place_name_for_trip TEXT NOT NULL DEFAULT 'Google';
```

Comment:
Why is this required at rider app end?
Isn't BPP returning the actual start and end location details?